design/sketch.webidl

```diff
@@ -195,18 +195,16 @@ interface WebGPUPipelineLayout {
 };
 
 // BindGroup
-dictionary WebGPUBufferBinding {
-    WebGPUBuffer buffer;
-    u32 offset;
-    u32 size;
-};
+dictionary WebGPUBinding {
```

Comment:
Why is this a dictionary?

Reply:
Would it be an interface instead? I don't see why.

Reply:
Yeah. An interface seems like a better fit.